Description: Bump up the maxsize on a fixed-size C buffer to avoid breaking on some autogenerated rust packages. (Closes: #942487)
Author: Ximin Luo <infinity0@debian.org>
Bug-Debian: https://bugs.debian.org/942487

--- reprepro-5.3.0.orig/indexfile.c
+++ reprepro-5.3.0/indexfile.c
@@ -63,7 +63,7 @@ retvalue indexfile_open(struct indexfile
 	f->linenumber = 0;
 	f->startlinenumber = 0;
 	f->status = RET_OK;
-	f->size = 256*1024;
+	f->size = 4*1024*1024;
 	f->ofs = 0;
 	f->content = 0;
 	/* +1 for *d = '\0' in eof case */
